app/i18n/locales/en-US.json

```diff
@@ -307,6 +307,9 @@
   "wallet.hw.ledger.common.error.105": "Network error. Please check your internet connection.",
   "wallet.navigation.receive": "Receive",
   "wallet.navigation.send": "Send",
+  "wallet.navigation.stakeDashboard": "Dashboard",
+  "wallet.navigation.stakeAdvancedSimulator": "Advanced staking simulator",
```

Comment:
@sarveshwar-singh they should be capitalized, also can you change the text to:

`Advanced Delegation Center`
`Simple Delegation Center`

Reply:
@nicarq 
Thanks, will update in upcoming PR.